src/gqlHooks/useDeleteFavoriteMovie.ts

```diff
@@ -0,0 +1,33 @@
+import { useMutation, gql } from "@apollo/client";
+import { GET_FAVORITE_MOVIES } from "./useGetFavoriteMovies";
+import { MOVIES_PER_PAGE } from "@components/constants";
+
+const DELETE_FAVORITE_MOVIE = gql`
+  mutation deleteFavoriteMovie($favMovieID: ID!) {
+    deleteFavoriteMovie(favMovieID: $favMovieID)
+  }
+`;
+
+export function useDeleteFavoriteMovie(page: number) {
+  const [deleteFavoriteMovie] = useMutation(DELETE_FAVORITE_MOVIE, {
+    refetchQueries: [
+      {
+        query: GET_FAVORITE_MOVIES,
+        variables: { page: page, moviesPerPage: MOVIES_PER_PAGE },
```

Comment:
очень страно, что useDeleteFavoriteMovie должен знать номер страницы для GET_FAVORITE_MOVIES 
если тебе придется использовать этот хук в другом месте, то возникнут проблемы
```suggestion
        variables: { page: page, moviesPerPage: MOVIES_PER_PAGE }, 
```
лучше тут попробовать использовать refetch из хука для GET_FAVORITE_MOVIES, refetch помнит параметры